packages/help-center/src/hooks/use-is-wapuu-enabled.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-restricted-imports */
+import config from '@automattic/calypso-config';
+import { useSelector } from 'calypso/state';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { useSupportAvailability } from '../data/use-support-availability';
+
+/**
+ * Helper hook to determine if Wapuu should be enabled.
+ * @returns {boolean} Whether Wapuu should be enabled or not.
+ * @example
+ * const isWapuuEnabled = useIsWapuuEnabled();
+ * if ( isWapuuEnabled ) {
+ *     // Do something
+ * }
+ */
+export const useIsWapuuEnabled = () => {
```

Comment:
The conditions should still be discussed, plus this will include the experiment and control treatments.